lib/pitchfork/chunked.rb

```diff
@@ -104,7 +104,9 @@ def chunkable_version?(ver)
     def call(env)
       status, headers, body = response = @app.call(env)
 
-      if chunkable_version?(env[Rack::SERVER_PROTOCOL]) &&
+      if !env.key?('rack.hijack_io') && # full highjack
+         !headers['rack.hijack'] && # partial hijack
```

Comment:
@byroot it should be noted that in Rack 2, headers is not guaranteed to be a hash. This was only specified in Rack 3. Previously, it only had to respond to `#each` IIRC.

Reply:
There are already checks later down in this middleware that do the exact same thing so I guess it may be fine to leave as is?

Reply:
While it's common to use hash as response headers, it's not a requirement of the Rack 2 spec. It is a requirement of the Rack 3 spec. I suppose if it's working, it's good enough?